### PR TITLE
vlc: add rtsp support

### PR DESCRIPTION
**Pull request by dutsky**

### CODE REVIEW

File: srcpkgs/live555/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Template file for 'live555'
+pkgname=live555
+version=2018.12.14
+revision=1
+wrksrc=live
+build_style=gnu-makefile
+short_desc="Live555 - set of C++ libraries for multimedia streaming, using open
```
Comment (maxice8) [inline srcpkgs/live555/template]: longer than 80 chars

**Comment by maxice8** [comment]
Cross is broken

File: srcpkgs/live555/template [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ standard protocols (RTP/RTCP, RTSP, SIP)"
+maintainer="Denis Revin <denis.revin@gmail.com>"
+license="LGPL-3.0-or-later"
+homepage="http://www.live555.com/liveMedia/"
+distfiles="http://www.live555.com/liveMedia/public/${pkgname%555}.$version.tar.gz"
+checksum=8dc31f76909602c3c07c8f876237e6ab7ac674d00f1bb065eda20e719f772a47
+
+pre_build() {
+	cd $wrksrc
```
Comment (Chocimier) [inline srcpkgs/live555/template]: pre_build is executed in wrksrc

File: srcpkgs/live555/template [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+version=2018.12.14
+revision=1
+wrksrc=live
+build_style=gnu-makefile
+short_desc="Live555 - set of C++ libraries for multimedia streaming, using open
+ standard protocols (RTP/RTCP, RTSP, SIP)"
+maintainer="Denis Revin <denis.revin@gmail.com>"
+license="LGPL-3.0-or-later"
+homepage="http://www.live555.com/liveMedia/"
+distfiles="http://www.live555.com/liveMedia/public/${pkgname%555}.$version.tar.gz"
```
Comment (Chocimier) [inline srcpkgs/live555/template]: please replace ${pkgname%555} with live

**Comment by dutsky** [comment]
Thank you for comments. Unfortunately vlc don't cross-building anyway, and I can't understand why, atm :(

**Comment by dutsky** [comment]
all fine

**Comment by Chocimier** [comment]
short_desc broken into two lines is worse than too long one. `Libraries for multimedia streaming using RTP/RTCP, RTSP, SIP - development` is longer than 72 characters specified by Manual, but could go IMO.

**Comment by dutsky** [comment]
short_desc fixed

File: srcpkgs/live555/template [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+		-D_FILE_OFFSET_BITS=64 -fPIC"
+fi
+
+pre_build() {
+	./genMakefiles linux-with-shared-libraries
+}
+
+live555-devel_package() {
+	short_desc+=" - development"
+	depends="${makedepends} ${sourcepkg}>=${version}_${revision}"
```
Comment (maxice8) [inline srcpkgs/live555/template]: There are no makedepends= defined

File: srcpkgs/live555/template [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+		-DSOCKLEN_T=socklen_t -D_LARGEFILE_SOURCE=1\
+		-D_FILE_OFFSET_BITS=64 -fPIC"
+fi
+
+pre_build() {
+	./genMakefiles linux-with-shared-libraries
+}
+
+live555-devel_package() {
+	short_desc+=" - development"
```
Comment (maxice8) [inline srcpkgs/live555/template]: development files

**Pull request closed without merging**